### ISSUE TITLE
[BUG buildall script does not support JDK11 profile

### ISSUE THREAD

**Issue by jlowe** [NVIDIA/spark-rapids#8690]
**Describe the bug**
Using the build/buildall script with the JDK11 profile produces the following error:
```
[INFO] Scanning for projects...
[ERROR] [ERROR] Could not find the selected project in the reactor: sql-plugin @ 
[ERROR] Could not find the selected project in the reactor: sql-plugin -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
```

**Steps/Code to reproduce bug**
`build/buildall -o=-Pjdk11`

**Comment by pxLi** [comment]
The buildall script works fine in my local ENV (using latest branch-23.08) using `build/buildall -o=-Pjdk11`

```
 % java --version
openjdk 11.0.19 2023-04-18
OpenJDK Runtime Environment (build 11.0.19+7-post-Ubuntu-0ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.19+7-post-Ubuntu-0ubuntu118.04.1, mixed mode, sharing)

% echo $JAVA_HOME
/usr/lib/jvm/java-11-openjdk-amd64
```

maybe there is some path error in your ENV

**Comment by jlowe** [comment]
This was fixed by #8711.  Thanks @gerashegalov!
